src/commands/transaction/multisigmodification.ts

```diff
@@ -62,6 +64,18 @@ export class CommandOptions extends AnnounceTransactionsOptions {
         description: 'Multisig account public key.',
     })
     multisigAccountPublicKey: string;
+
+    @option({
+        flag: 't',
+        description: 'Aggregate Type (complete or bonded)',
+    })
+    aggregateType: string;
+
+    @option({
+        flag: 'd',
+        description: 'Deadline (milliseconds, without Nemesis epoch adjustment)',
```

Comment:
Now that I think about it, I don't know what this deadline is. Is this milliseconds from now? Or from epoch? From nemesis?
Explain it to me and I'll suggest a text.

Reply:
> The deadline of the transaction. The deadline is given as the number of seconds elapsed since the creation of the nemesis block. If a transaction does not get included in a block before the deadline is reached, it is deleted.

https://docs.symbolplatform.com/symbol-sdk-typescript-javascript/0.19.2/classes/_model_transaction_deadline_.deadline.html